client/public/js/whiteboard.js

```diff
@@ -19,48 +30,58 @@ const newCanvas = function(){
 }
 
 const drawTouch = function() {
-	let start = function(e) {
-		ctx.beginPath();
-		x = e.changedTouches[0].pageX;
-		y = e.changedTouches[0].pageY - 34;
-		ctx.moveTo(x, y);
+
+	const startTouch = function(e) {
+    // if (e.targetTouches.length < 2) {
+      // e.preventDefault();
+  		ctx.beginPath();
+  		x = e.changedTouches[0].pageX + content.scrollLeft;
+  		y = e.changedTouches[0].pageY - 34 + content.scrollTop;
+  		ctx.moveTo(x, y);
+    // }
 	};
-	let move = function(e) {
-		e.preventDefault();
-		x = e.changedTouches[0].pageX;
-		y = e.changedTouches[0].pageY - 34;
-		ctx.lineTo(x, y);
-		ctx.stroke();
+	const moveTouch = function(e) {
+    // if (e.targetTouches.length < 2) {
+    if (canvasState) {
+      e.preventDefault();
+      x = e.changedTouches[0].pageX + content.scrollLeft;
+      y = e.changedTouches[0].pageY - 34 + content.scrollTop;
+      ctx.lineTo(x, y);
+      ctx.stroke();
+    }
+    // }
 	};
-  document.getElementById('canvas').addEventListener('touchstart', start, false);
-	document.getElementById('canvas').addEventListener('touchmove', move, false);
+
+  canvas.addEventListener('touchstart', startTouch, false);
+	canvas.addEventListener('touchmove', moveTouch, false);
```

Comment:
I believe `canvas` here is undefined unless I'm missing a global variable somewhere